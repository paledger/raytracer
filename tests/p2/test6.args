pixelray ../resources/simp_cam1.pov 640 480 320 240
