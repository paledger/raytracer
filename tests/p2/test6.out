Pixel: [320, 240] Ray: {-14 0 0} -> {1 0.001042 0.001042}
